ranger/core/main.py

```diff
@@ -407,8 +407,14 @@ def load_custom_commands(*paths):
         except OSError:
             LOG.debug('Unable to access plugin directory: %s', plugindir)
         else:
-            plugins = [p[:-3] for p in plugin_files
-                       if p.endswith('.py') and not p.startswith('_')]
+            plugins = []
+            for p in plugin_files:
+                if not p.startswith('_'):
```

Comment:
What's this about exactly? I know you didn't add it but I don't understand and wouldn't it prevent the `__init__.py`s to be loaded from the subdirectories?

Reply:
`plugin_files` is a list of files/directories in `~/.config/ranger/plugins` and the `__init__.py` files are located within one of the directories, so they're not getting ignored.